lxd/container.go

```diff
@@ -412,6 +425,7 @@ type container interface {
 	LogPath() string
 
 	// FIXME: Those should be internal functions
+	StoragePool() string
```

Comment:
Fwiw, I also think that `StorageStart()` and `StorageStop()` are now appropriate.